tests/trainer/test_sharded_checkpoint.py

```diff
@@ -240,6 +165,48 @@ def test_fsdp_full_state_dict_load(world_size, tmp_path: pathlib.Path, autoresum
     trainer2.fit()
 
 
+@pytest.mark.gpu
+@pytest.mark.remote
```

Comment:
can u post a screenshot showing this passes as it only runs on daily?

Reply:
lol lets hope @eracah ran it, but yeah, will run and include

Reply:
done